src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -801,6 +801,10 @@ public User createUser(User user) {
         return complete(createUsersAsync(users));
     }
 
+    public JobStatus<User> createOrUpdateUsers(List<User> users) {
```

Comment:
It might be useful to add varargs version like for `createUsers`:

```
    public JobStatus<User> createOrUpdateUsers(User... users) {
        return createOrUpdateUsers(Arrays.asList(users));
    }
```